js/packages/openinference-instrumentation-llama-index/src/instrumentation.ts

```diff
@@ -56,17 +63,21 @@ function getExecContext(span: Span) {
   }
   return execContext;
 }
-export class LlamaIndexInstrumentation extends InstrumentationBase<
-  typeof llamaindex
-> {
+
+export const RETRIEVAL_DOCUMENTS =
```

Comment:
if this isn't exported from semantic conventions and you need it you can add it, I think it might already be exported